### PR TITLE
Rethinked the bundle configuration

### PR DESCRIPTION
**Pull request by dunglas**
And added a test for changing the number of item by page client side.

Ping @amenophis

### CODE REVIEW

File: Controller/ResourceController.php [hunk 1]
```diff
@@ -122,16 +122,16 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
      */
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
-        $page = (int) $request->get('page', 1);
+        $page = (int) $request->get($this->container->getParameter('api.collection.pagination.page_parameter_name'), 1);
 
-        $itemsPerPage = $this->container->getParameter('api.default.items_per_page');
+        $defaultItemsPerPage = $this->container->getParameter('api.collection.pagination.items_per_page.number');
```
Comment (amenophis) [inline Controller/ResourceController.php]: why number ?

File: features/collection.feature [hunk 2]
```diff
@@ -169,6 +169,138 @@ Feature: Collections support
...
+  Scenario: Change the number of element by page client side
+    When I send a "GET" request to "/dummies?page=2&itemsPerPage=10"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json"
+    And the JSON should be equal to:
+    """
+    {
+      "@context": "/contexts/Dummy",
+      "@id": "/dummies/dummies?page=2",
```
Comment (amenophis) [inline features/collection.feature]: miss &itemsPerPage=10 in the @id, no ? page=2 alone mean nothing

**Comment by amenophis** [comment]
Where is the modification for test about kernel environment configuration ?

File: Controller/ResourceController.php (hunk 1 above)
Comment (dunglas) [inline Controller/ResourceController.php]: The default number of element by pages.

File: features/collection.feature (hunk 2 above)
Comment (dunglas) [inline features/collection.feature]: Good catch!

**Comment by dunglas** [comment]
Not done yet.

**Pull request merged**